src/main/resources/init.sql

```diff
@@ -0,0 +1 @@
+CREATE SCHEMA IF NOT EXISTS accruals;
```

Comment:
Not sure if this is the right place. The schema needs to be set as the liquibase container needs the schema before this container is started see `--liquibaseSchemaName`
Looks like schema creation is already covered here
https://github.com/UKHomeOffice/callisto-localdev/blob/7217450491bc3c7382f4c839c3ba1676fa59331a/docker-compose.yml#L214

Reply:
This is only used for the new integration tests.
We are looking at trying to get the final solution to be integrated with liquibase. So hopefully we can remove this. But that is a WIP.